app/assets/javascripts/components/widget-rss.js

```diff
@@ -0,0 +1,38 @@
+ManageIQ.angular.app.component('widgetRss', {
+  bindings: {
+    id: '<',
+  },
+  controllerAs: 'vm',
+  controller: ['$http', 'miqService', '$sce', function($http, miqService, $sce) {
+    var vm = this;
+    vm.widgetRssModel = {};
+
+    this.$onInit = function() {
+      $http.get('/dashboard/widget_rss_data/' + vm.id)
+        .then(function(response) { vm.widgetRssModel.content = $sce.getTrustedHtml(response.data.content); })
```

Comment:
Here, you do need trustAsHtml, otherwise, the entries are not clickable anymore..